tests/test_rpc.py

```diff
@@ -36,6 +36,24 @@ def test_getaddress(lianad):
     assert res["address"] != lianad.rpc.getnewaddress()["address"]
 
 
+def test_listadresses(lianad):
```

Comment:
I think you still need to apply [black](https://github.com/psf/black) formatting to this test.

Reply:
yep, done ;)